src/rpc/blockchain.cpp

```diff
@@ -432,7 +432,9 @@ static UniValue getdifficulty(const JSONRPCRequest& request)
 
 static std::string EntryDescriptionString()
 {
-    return "    \"size\" : n,                 (numeric) transaction size in bytes\n"
+    return "    \"vsize\" : n,                (numeric) transaction size in bytes\n"
```

Comment:
I feel like we should just drop 15637. We don't really have vsize, and not sure it makes sense to add it

Reply:
Note: We kind of have vsize internally now after #4562 but we don't have it in any rpcs, so 15637 is still a DNM probably

Reply:
Dropped 15637